packages/api/src/controllers/stripe.ts

```diff
@@ -214,6 +225,70 @@ const sendUsageRecordToStripe = async (
   );
 };
 
+export const calculateOverageOnMinimumSpend = async (product, billingUsage) => {
+  const minimumSpend = product.monthlyPrice;
```

Comment:
Did you mean `const minimumSpend = product.minimumSpend`? Otherwise please either rename the variable or add a comment to explain the concept change here

Reply:
minimumSpend is a boolean and monthlyPrice is the actual minimumSpend per month. Agree that I could give a better naming, will give a think